jsonrpc/types.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/0xPolygon/polygon-edge/types"
 )
 
+const jsonRPCMetric = "json_rpc"
```

Comment:
Just thinking out loud: maybe we should keep all metrics prefixes in a single place?

Reply:
They are package related, so I don't think it will bring much benefit to have them in single place, but whatever we agree upon.